README.yaml

```diff
@@ -43,3 +43,5 @@ contributors:
     github: "goruha"
   - name: "Andriy Knysh"
     github: "aknysh"
+  - name: "Benjamin Smith"
+    github: "Benbentwo"
```

Comment:
FYI this doesnt do anything. Contributors are pulled from `contribs`. But doesnt hurt either

Reply:
(with the updated README template)

Reply:
Yeah added mostly to trigger a build